config/etfuturum/blocksitems.cfg

```diff
@@ -0,0 +1,445 @@
+
+    # If this is disabled, netherite items will not be craftable unless added by CraftTweaker. This introduces the smithing GUI from versions prior to 1.20. It is compatible with CraftTweaker.
+    # The mod prefix is "mods.etfuturum.smithingTable", and the functions are "addRecipe" or "addRecipeNoNBT". "addRecipe" will copy all NBT data from the first slot to the output. (and damage if the first slot and output are damageable items) You can remove recipes using the "remove" function and an ItemStack.
+    # Currently CraftTweaker support is limited, so some features from CraftTweaker may not work. .noReturn() and .anyDamage().onlyDamaged() are known to not work.
+    # However anyDamage() by itself does work, and is required for tool inputs to actually copy their damage over to the output.
+    # Examples:
+    # mods.etfuturum.smithingTable.addRecipe(<IC2:itemToolBronzeSword>, <minecraft:iron_sword>.anyDamage(), <ore:ingotBronze>); //(Use "ingotBronze" on iron sword to convert it to a bronze sword)
+    # mods.etfuturum.smithingTable.addRecipeNoNBT(<etfuturum:sponge:1>, <minecraft:sponge>, <minecraft:water_bucket>); //Take a vanilla sponge and a water bucket, you will get an Et Futurum Requiem wet sponge. [default: true]
+    B:enableSmithingTable=true
+
+    # Will attempt to seek and auto-add recipes to itself. Look at ConfigFunctions.cfg "autoAddSmoker" for more info.
+    # Compatible with CraftTweaker. In the same way that you'd use furnace.addRecipe or furnace.remove, you can use "mods.etfuturum.smoker" instead of "furnace". [default: true]
+    B:enableSmoker=true
+
+    #  [default: true]
+    B:enableSponge=true
+
+    # Currently DOES NOT HAVE ANY FUNCTIONALITY. Decoration ONLY! [default: true]
+    B:enableStonecutter=true
+
+    # Enables target block from 1.16 [default: true]
+    B:enableTarget=true
+
+    # Enables wood variant trapdoors [default: true]
+    B:enableTrapdoors=false
+
+    # Enables wood variant buttons and pressure plates [default: true]
+    B:enableWoodRedstone=true
+}
+
+
+##########################################################################################################
+# misc blocks
+#--------------------------------------------------------------------------------------------------------#
+# Blocks that don't fit in any other category.
+##########################################################################################################
+
+"misc blocks" {
+    #  [default: true]
+    B:enableBanners=true
+
+    # Enables log blocks with bark on all sides [default: true]
+    B:enableBarkLogs=true
+
+    # A solid, indestructible and invisible block. Can be seen when holding it in Creative mode. [default: true]
+    B:enableBarrier=true
+
+    # This must be on for the basalt deltas biome to generate unless Netherlicious is installed. [default: true]
+    B:enableBlackstone=true
+
+    #  [default: true]
+    B:enableChain=true
+
+    #  [default: true]
+    B:enableConcrete=true
```

Comment:
we already have hempcrete

Reply:
I was going to make the recipes match hempcrete rather than disable it since this gives us some different color options.

Reply:
they have different colors?

Reply:
They're a bit different than the hempcrete colors. Not a huge difference. I'm happy to disable this if you think it's not worth the trouble.

Reply:
yea for a slight hue difference that does not sound worth doubling the blocks.

Reply:
Disabled.